ui/src/views/Tasks/ViewTask/index.jsx

```diff
@@ -747,7 +746,9 @@ export default class ViewTask extends Component {
                 }
               />
             ))}
-            <Divider className={classes.divider} />
+            <br />
```

Comment:
The first `<br />` doesn't seem to do anything since it's being inserted beside the chip.

![Screen Shot 2019-10-07 at 3 44 53 PM](https://user-images.githubusercontent.com/3766511/66343355-7bae8580-e919-11e9-9ce5-5591434943ed.png)

To fix this issue, we can wrap the all the `<Chip />` with a `<div>` and then only insert two `<br />` as opposed to three.

```jsx
<div>
  <Chip ... />
  {tags.map()}
</div>
<br />
<br />
<Grid .. />
```

Reply:
Thank you @helfi92 and apologies for my delayed progress here. I just finished a work contract and moved states so it's been a busy couple weeks! I made that fix in the commit below, please review and let me know any feedback :) just curious- why a`<div>`instead of a `<Fragment>` ? I thought a `<Fragment>`  might work best for wrapping the `<Chip/>`

Reply:
Actually from looking back, I think it's because I need to move the key attribute from my `<Chip/>` to my `<div>`. Will make those changes and also need to the latest master!